code/modules/mob/mob.dm

```diff
@@ -903,13 +903,8 @@ var/list/slot_equipment_priority = list( \
 /mob/proc/can_use_hands()
 	return
 
-/mob/proc/is_active()
-	return (0 >= usr.stat)
-
 /mob/proc/is_mechanical()
-	if(mind && (mind.assigned_role == "Cyborg" || mind.assigned_role == "AI"))
-		return 1
-	return istype(src, /mob/living/silicon) || get_species() == "Machine"
```

Comment:
That's handled by the `(species.flags & ALL_RPARTS)` in the human-level implementation of this proc.